docs/howtoguides/get_token_and_attach.md

```diff
@@ -35,3 +46,24 @@ Enable services with: pro enable <service>
 Once the Ubuntu Pro Client is attached to your Ubuntu Pro account, you can use
 it to activate various services, including: access to ESM packages, Livepatch,
 FIPS, and CIS. Some features are specific to certain LTS releases.
+
+## Control of auto-enabled services
+
+Your subscription controls which services are available to you as shown and
+controlled on the [Ubuntu Pro Dashboard](https://ubuntu.com/pro/dashboard).
+
+Recommended services are auto-enabled when attaching a system. Which services
+are considered recommended can, for some subscriptions, be controlled on the
```

Comment:
I'm not sure about this bit here. The wording implies that you're toggling which services are recommended, but I was under the impression there are certain services that we recommend (which are auto-enabled) and the toggle just allows you to enable/disable those. If the latter is the case, let me know and I'll tighten up the wording.

Reply:
The initial setting is what "we recommend", but you can toggle it to make your own selection.
Indeed the use of the word recommended is misleading and I only used it because that is how they were called before.

So the flow is 1. you setup a subscription -> 2. you go to dashboard, recommended services are selected by default -> 3. you can adjust there as/if you want -> 4. you attach and those enabled will be auto enabled.

Reply:
That's what I thought - I'll recommend an alternative